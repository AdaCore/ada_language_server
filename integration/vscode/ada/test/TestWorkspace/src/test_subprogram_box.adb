procedure Test_Subprogram_Box is

    procedure Do_Nothing is
    begin
        null;
    end Do_Nothing;

    task body T is
    begin
        null;
    end T;
begin
    null;
end Test_Subprogram_Box;