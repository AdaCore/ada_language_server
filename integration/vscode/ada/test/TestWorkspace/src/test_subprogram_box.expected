-------------------------
-- Test_Subprogram_Box --
-------------------------

procedure Test_Subprogram_Box is

    ----------------
    -- Do_Nothing --
    ----------------

    procedure Do_Nothing is
    begin
        null;
    end Do_Nothing;

    -------
    -- T --
    -------

    task body T is
    begin
        null;
    end T;
begin
    null;
end Test_Subprogram_Box;